Intersect.Client/Entities/Player.cs

```diff
@@ -1454,8 +1454,22 @@ private void SetTargetBox(Entity en)
 
         private void AutoTurnToTarget(Entity en)
         {
-            if (!Globals.Database.AutoTurnToTarget || !Options.Instance.PlayerOpts.EnableAutoTurnToTarget ||
-                Controls.KeyDown(Control.TurnAround))
+            if (!Options.Instance.PlayerOpts.EnableAutoTurnToTarget)
+            {
+                return;
+            }
+
+            if (!Globals.Database.AutoTurnToTarget)
+            {
+                return;
+            }
+
+            if (Controls.KeyDown(Control.TurnAround))
+            {
+                return;
+            }
+
+            if (en == this)
```

Comment:
Re-order these checks:

1. `en`, because it's the closest value to check since it's a parameter
2. `Globals.Database.AutoTurnToTarget` (1-2 levels of pointer indirection depending on how the compiler handles statics like Database`)
3. `Options.Instance.PlayerOpts.EnableAutoTurnToTarget` (2-3 levels of pointer indirection)
4. `Controls.KeyDown(Control.TurnAround)` (this is a method call, on top of pointer indirection)

Reply:
Done 👍🏽